src/components/Calculator.js

```diff
@@ -5,36 +5,36 @@ export default class Calculator extends React.Component {
   render() {
```

Comment:
- [ ] The Calculator component should import the calculate.js file from the logic directory, this means that your logic will need to change a little bit to adapt to this change. You will most likely need to move some content from your App.js script to the Calculator.js component, please make these changes to comply with the project requirements.